### ISSUE TITLE
Allow seeking while playing a live HLS stream

### ISSUE THREAD

**Issue by kolpax** [clappr/clappr#871]
**Browser**: Chrome Version 48.0.2564.116 (64-bit)

**OS**: OSX 10.11.3 (15D21)

**Clappr Version**: 0.2.38

**Steps to reproduce**:

1. Load a source that is a live HLS playlist file (.m3u8).
2. Wait for the video to start and show the `live ·` icon.
3. Seek backwards in the video.

I would expect to be able to seek from the first chunk in the playlist file to the latest chunk but I cannot seek at all.

**Comment by leandromoreira** [comment]
@kolpax in fact you are only able to seek back when you reach more a threshold (which I believe it's 60s)

**Comment by ronag** [comment]
@leandromoreira: Why is there such a limit?

**Comment by ronag** [comment]
I've tried with a 2 minute clip and seeking live HLS is not allowed.

**Comment by leandromoreira** [comment]
@ronag we thought it'd be better to only offer the seek back feature when a threshold was achieved. 
But you can configure it either on [hls (hls.js)](https://github.com/clappr/clappr/blob/master/src/playbacks/hls/hls.js#L33) or [flashls](https://github.com/clappr/clappr/blob/master/src/playbacks/flashls/flashls.js#L113)

**Comment by leandromoreira** [comment]
It's working fine here :disappointed:

**Comment by ronag** [comment]
@leandromoreira: How do I pass down those options?

**Comment by ronag** [comment]
Tried: 

```javascript

    this.player = new clappr.Player({
      parent: div,
      width: '100%',
      height: '100%',
      hlsMinimumDvrSize: 0,
    })
```

No success. Can't seek.

**Comment by tjenkinson** [comment]
@ronag I think the reason as well for the threshold is because of the nature of HLS. Live streams without dvr and live streams with dvr are essentially the same thing.

A stream that doesn't have dvr may have a playlist containing a few chunks, maybe 10 seconds. If there was no threshold and we always showed the seek bar it might look a bit strange for a 10 second window of content.

`hlsMinimumDvrSize` set to 0 should definately work.

Can you right click on the video and inspect it, and see what the `src` of the video tag is?

It should be something like `blob:http%3A//cdn.clappr.io/a4952826-c0be-4ab5-a08d-4eb02f221288` if hlsjs is being used.

**Comment by ronag** [comment]
Could you try with this?

https://www.dropbox.com/sh/tmugpsriq21s4zh/AAASc9JyKjHAbrYhbZEOz0jRa?dl=0

It's definitively using HLSJS tried debugging `onLevelUpdated` and see whether it gets a duration and it does...

**Comment by tjenkinson** [comment]
Thanks for the files.

When I said dvr I basically just mean recording. So some live streams might not have a recording, whereas some might let people rewind the past half hour, or some might let people always rewind back to the start of the stream (which fits the EVENT type). In live streams which don't have a recording, they still have a few fragments in their playlist, and therefore a small recording, but it doesn't make sense to let the user seek in this small recording.

So I have found the bug and will submit a fix shortly. At the moment the type of the media has to be "live" and the duration longer than the one provided for seek to be enabled. Seek should always be enabled if the type is vod and at the moment this isn't the case. When it's fixed you don't need to provide `hlsMinimumDvrSize` as that only applies to live streams.

(Also I tweaked the urls in the playlist file to 'test.ts')
